### ISSUE TITLE
`update` should not be `type: Any` based on description

### ISSUE THREAD

**Issue by jeremyfiel** [OAI/Overlay-Specification#70]
https://github.com/OAI/Overlay-Specification/blob/d029f0d7d2c8861e00de4467ab795e86d9e35824/versions/1.0.0.md?plain=1#L125

This text indicates the `type` of `update` should only be `object | array` and _not_ `Any`

**Comment by lornajane** [comment]
Good catch! In this case though, we know that there are already implementations built from the spec, and we aim to formalise the existing content. We decided against making things more restrictive (but agree this could be tightened for a future release)
